MetaMask/metamask-mobile
# feat: display "Snaps (Beta)" decorator tag in accounts list

## **Description**

This PR adds a "Snaps (Beta)" decorator tag for Snap accounts in the mobile accounts list. 
The changes include:

- Modified the AccountSelectorList component to identify Snap accounts.
- Added a new localization string for the "Snaps (Beta)" tag.
- Updated the rendering logic to display the tag for Snap accounts.

## **Related issues**

Fixes: https://github.com/MetaMask/accounts-planning/issues/150

## **Manual testing steps**

1. Go to the [SSK](https://metamask.github.io/snap-simple-keyring/latest/) and create an account
2. Open the accounts list 
3. You _should_ see the tag "Snaps (beta)" under the name "Snap Account x"
4. Create more accounts on the SSK
5. You _should_ see the snaps tag under each snap account

## **Screenshots/Recordings**

### **Before**

<img src="https://github.com/user-attachments/assets/f13a4a66-9f24-42d0-9f24-4faed16205d3" width=300 />

### **After**

<img src="https://github.com/user-attachments/assets/5065229e-b435-4f9d-a746-20521753ebf0" width=300 />

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
